src/components/chip-set/chip-set.tsx

```diff
@@ -466,6 +473,15 @@ export class ChipSet {
     }
 
     private isInvalid() {
+        if (this.readonly) {
+            // A readonly field can never be invalid.
+            return false;
+        }
+
+        if (this.invalid) {
```

Comment:
Can disabled field be invalid?

Reply:
Good question. I don't know. I don't see any logic handling this scenario in the input-field. So I guess it can be invalid yes. Probably it won't show anything in the UI when it's disabled && invalid. What do you think?

Reply:
seems like the disabled styling overrides the invalid styling, but I don't think that's something we need to be concerned with here.

If it's set as both invalid and disabled, I see no reason why `isInvalid()` shouldn't return true.

Reply:
This argument can be used here as well maybe?
> So when something is non-interactive and dead, the system cannot tell the user to fix it, because it is now invalid! Users cannot do anything about the validity of the field.

Reply:
Good find. I am convinced that setting something as both disabled an invalid is a mistake or the result of some inconsistent state.

Should we try to hide the mistake or inconsistency in lime-elements, without clear documentation?

Or is there some valid use case when a field has errors, and it should still be disabled? I guess it won't be possible to save a form then, and that would be even more confusing if we hide the error.

Reply:
I think if a field can be invalid, then it shouldn't be disabled then. Doesn't it look like a bad design, done by the consumer?

Should the invalid styles be applied to a disabled && invalid field? Yes. Because then we show them that they've done something stupid, or something is effed up somewhere along the way. I think it's good.

Note that `readonly` state is another story. A disabled field can become enabled, if user does something. But a readonly filed can never become enabled.